opencti-platform/opencti-front/src/private/components/common/files/PictureManagementEdition.tsx

```diff
@@ -0,0 +1,170 @@
+import { Theme } from '../../../../components/Theme';
+import { handleErrorInForm } from '../../../../relay/environment';
+import { pictureManagementUtilsMutation } from './PictureManagementUtils';
+import { PictureManagementUtils_node$data } from './__generated__/PictureManagementUtils_node.graphql';
+import { PictureManagementUtilsMutation } from './__generated__/PictureManagementUtilsMutation.graphql';
+
+const useStyles = makeStyles<Theme>((theme) => ({
+  header: {
+    backgroundColor: theme.palette.background.nav,
+    padding: '20px 20px 20px 60px',
+  },
+  closeButton: {
+    position: 'absolute',
+    top: 12,
+    left: 5,
+    color: 'inherit',
+  },
+  container: {
+    padding: '10px 20px 20px 20px',
+  },
+  appBar: {
+    width: '100%',
+    zIndex: theme.zIndex.drawer + 1,
+    borderBottom: '1px solid #5c5c5c',
+  },
+  title: {
+    float: 'left',
+  },
+  buttons: {
+    marginTop: 20,
+    textAlign: 'right',
+  },
+  button: {
+    marginLeft: theme.spacing(2),
+  },
+}));
+
+interface PictureManagementEditionProps {
+  picture: PictureManagementUtils_node$data
+  entityId: string
+  handleClose: () => void;
+}
+
+interface PictureManagementEditionFormValues {
+  id: string
+  description: string | null;
+  order: number | null;
+}
+
+const PictureManagementEdition: FunctionComponent<PictureManagementEditionProps> = ({
+  picture,
+  entityId,
+  handleClose,
+}) => {
+  const { t } = useFormatter();
+  const classes = useStyles();
+  const [commit] = useMutation<PictureManagementUtilsMutation>(pictureManagementUtilsMutation);
+  const pictureValidation = () => Yup.object().shape({
+    description: Yup.string().nullable(),
+    order: Yup.number().nullable().integer(t('The value must be a number')),
+  });
+  const onSubmit: FormikConfig<PictureManagementEditionFormValues>['onSubmit'] = (
+    values,
+    { setSubmitting, setErrors },
+  ) => {
+    const input: StixDomainObjectFileEditInput = {
+      id: values.id,
+      description: values.description,
+      order: values.order,
+    };
+    commit({
+      variables: {
+        id: entityId,
+        input,
+      },
+      onCompleted: () => {
+        setSubmitting(false);
+        handleClose();
+      },
+      onError: (error: Error) => {
+        handleErrorInForm(error, setErrors);
+        setSubmitting(false);
+      },
+    });
+  };
+
+  return (
+    <div>
+      <div className={classes.header}>
+        <IconButton
+          aria-label="Close"
+          className={classes.closeButton}
+          onClick={handleClose}
+          size="large"
+          color="primary"
+        >
+          <Close fontSize="small" color="primary" />
+        </IconButton>
+        <Typography variant="h6" classes={{ root: classes.title }}>
+          {t('Update a picture')}
+        </Typography>
+        <div className="clearfix" />
+      </div>
+      <div className={classes.container}>
+        <Formik
+          initialValues={{
+            id: picture.id,
```

Comment:
can the picture id change ? otherwise I don't think it should be in the form, we could directly use `picture.id`